packages/types/config/build.d.ts

```diff
@@ -95,6 +97,39 @@ interface Warning {
   name: string
 }
 
+interface PostcssOrderPresetFunctions {
+  cssnanoLast: (names: string[]) => string[]
+  presetEnvAndCssnanoLast: (names: string[]) => string[]
+  presetEnvLast: (names: string[]) => string[]
+}
+type PostcssOrderPreset = keyof PostcssOrderPresetFunctions
+interface PostcssVariableMap {
+  customMedia: Record<string, string>
```

Comment:
mandatory ? Or is it fine cause used later with `Partial` ?

Reply:
@kevinmarrec Yes, I've made it mandatory because it is passed to the `exportTo` function, and I've made it Partial wherever it is config. I wanted to indicate that `environmentVariables` may not exist because although it is referenced in the postcss-preset-env README, I can find no reference in the codebase.

Reply:
Ok good :)